src/DataWrangling/ECCO2.jl

```diff
@@ -11,10 +11,23 @@ using Oceananigans.Utils
 using KernelAbstractions: @kernel, @index
 using NCDatasets
 
+import Oceananigans.Fields: set!
+
 temperature_filename = "THETA.1440x720x50.19920102.nc"
 salinity_filename = "SALT.1440x720x50.19920102.nc"
 effective_ice_thickness_filename = "SIheff.1440x720.19920102.nc"
 
+# Ecco field used to set model's initial conditions
+struct ECCO2Field
```

Comment:
```suggestion
struct ECCO2FieldInformation
```

so it's not confused with `Field`

Reply:
or `ECCO2FieldDescription`